llama-index-integrations/indices/llama-index-indices-managed-llama-cloud/llama_index/indices/managed/llama_cloud/base.py

```diff
@@ -69,13 +69,19 @@ def __init__(
         callback_manager: Optional[CallbackManager] = None,
         httpx_client: Optional[httpx.Client] = None,
         async_httpx_client: Optional[httpx.AsyncClient] = None,
+        pipeline_id: Optional[str] = None,
+        index_id: Optional[str] = None,  # alias for pipeline_id
         **kwargs: Any,
     ) -> None:
         """Initialize the Platform Index."""
         self.name = name
         self.project_name = project_name
         self.organization_id = organization_id
         self.transformations = transformations or []
+        self._pipeline_id = index_id or pipeline_id
+
+        if self._pipeline_id is None and self.name is None:
```

Comment:
no check for index id

Reply:
➕

Reply:
index id is already assigned to `self._pipeline_id`?